components/meta-packages/sunray-essential/sunray-essential.p5m

```diff
@@ -0,0 +1,107 @@
+# empty package only for satisfy installation requirements
+# SRSS on Solaris uses the old Sun DHCP server but on OI we use ISC-DHCP
+depend fmri=service/network/dhcp type=require
+
+# SUNWutkio-libs package dependency, empty packages only
+# for satisfy installation requirements
+depend fmri=consolidation/X/X-incorporation type=require
+depend fmri=compatibility/packages/SUNWxwplt type=require
+
+# for Sun JRE
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libCrun.so.1
+
+# SUNWut configure requirements rpm2cpio, see docs.openindiana.org
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/bin/rpm2cpio
+
+# in OI legacy/retired 64bit packages
+# GDM and Gnome greeter, only gdm-session-worker has to run in 32bit
+#
+# pkg.debug.depend.file does not working yet, there is no appropriate package in the OI repo
+# depend type=require fmri=__TBD pkg.debug.depend.file=usr/sbin/gdm-binary
+depend type=require fmri=system/display-manager/gdm@2.30.7
+# depend type=require fmri=__TBD pkg.debug.depend.file=usr/bin/gnome-session-properties
+depend fmri=gnome/gnome-session@2.32.1 type=require
+# depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/amd64/libgnome-desktop-3.so
+depend fmri=library/desktop/gnome-desktop@2.32.1 type=require
+# depend type=require fmri=__TBD pkg.debug.depend.file=/usr/libexec/gnome-settings-daemon
+depend fmri=gnome/gnome-settings-daemon@2.32.1 type=require
+depend fmri=library/gnome/libgnomekbd@2.91.92 type=require
+
+# 32bit packages
+# only for loginGUI
+depend fmri=cde/cde-runtime@0.5.11 type=require
+depend fmri=library/tooltalk@0.5.11 type=require
+depend fmri=library/motif@0.5.11 type=require
+
+# gdm-session-worker 32bit libraries
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libglib-2.0.so.0
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/mps/libnspr4.so
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/mps/libnss3.so
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libdbus-glib-1.so.2
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libdbus-1.so.3
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libpcre2-8.so.0
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libffi.so.6
+
+# utaudio 32bit libs
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libpulse.so.0
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libmpg123.so.0
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libFLAC.so.14
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libsndfile.so.1
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libSDL2-2.0.so.0
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libvorbis.so.0
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libopus.so.0
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libogg.so.0
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libmp3lame.so
+
+# uttsc-bin (RDP client)
```

Comment:
Could you please elaborate what exactly this means?

Reply:
These are libraries which are used by uttsc-bin, the SunRay RDP-client